### PR TITLE
[CLC-429]: Change cancel message on dmt start command

### PR DESCRIPTION
**Pull request by kutluhanmetin**

### CODE REVIEW

File: base/commands/migration/migration_start.go [hunk 1]
```diff
@@ -95,5 +98,6 @@ Selected data structures in the source cluster will be migrated to the target cl
 }
 
 func init() {
+	cmd.SetCancelMsg(" (Ctrl+C to exit) ")
```
Comment (yuce) [inline base/commands/migration/migration_start.go:101]: It's better to put this at the start of `Exec`, so it doesn't affect other commands.

**Review by yuce — CHANGES_REQUESTED** [review]
Just one thing...

File: base/commands/migration/migration_start.go (hunk 1 above)
Comment (kutluhanmetin) [reply base/commands/migration/migration_start.go]: https://github.com/hazelcast/hazelcast-commandline-client/pull/431/commits/58740c1075225cd14bf5664bd48909fb90f4ce80

File: base/commands/migration/migration_start.go [hunk 2]
```diff
@@ -41,7 +43,9 @@ func (StartCmd) Exec(ctx context.Context, ec plug.ExecContext) (err error) {
 	ec.PrintlnUnnecessary(`Hazelcast Data Migration Tool v5.3.0
 (c) 2023 Hazelcast, Inc.
 	
-Selected data structures in the source cluster will be migrated to the target cluster.	
+Selected data structures in the source cluster will be migrated to the target cluster.
+
+(Pressing CTRL+C does not cancel the migration progress)
```
Comment (yuce) [inline base/commands/migration/migration_start.go:48]:
```suggestion
Exiting the application does not affect the migration progress.
In order to cancel the migration, use the 'cancel' command.
```

**Review by yuce — CHANGES_REQUESTED** [review]
One idea only...

File: base/commands/migration/migration_start.go (hunk 2 above)
Comment (kutluhanmetin) [reply base/commands/migration/migration_start.go]: https://github.com/hazelcast/hazelcast-commandline-client/pull/431/commits/6a4d04e1beb26deada69113a3db477a0afde7c10

**Review by yuce — APPROVED** [review]

**Review by srknzl — APPROVED** [review]

**Pull request merged**